src/stateOfDevopsDashboardGenerator.ts

```diff
@@ -382,7 +385,14 @@ export class StateOfDevOpsDashboardGenerator {
                 if (data === null) {
                     resolve(state);
                 } else {
-                    state.pipelineNames = data.pipelines.map((m: any) => m.name);
+                    state.pipelineNames = data.pipelines.map((m: any) => {
+
+                        if (appNames.some(substring => m.name.includes(substring))) { return m.name }
+
+                    })
+                    state.pipelineNames = state.pipelineNames.filter(function (element: any) {
```

Comment:
Why is your map returning undefined?

Reply:
The data passed in can have possible undefined values. Hence the map might return undefined. I dont think the filter function actually hurts.